draft-ietf-jsonpath-base.md

```diff
@@ -579,8 +559,8 @@ root-selector  = "$"
 #### Semantics
 {: unnumbered}
 
-The Argument — the root JSON value — becomes the root node, which is
-addressed by the root selector `$`.
+The root selector `$` selects the root node of the argument
+and produces a nodelist consisting of that root node.
```

Comment:
Is the result of the root selector always a nodelist?

Reply:
Yes.